### DISCUSSION TITLE
World time clock widget

### DISCUSSION THREAD

**Discussion by bilogic** [filamentphp/filament#9818] [Help]
https://www.worldtimebuddy.com/

In the spirit of code re-use, has anyone come across any filament widget similar to world time buddy, which allows me to quickly convert any date/time between timezones

Thank you.

**Comment by sinnbeck** [comment]
You could just copy paste the code into a custom widget?

https://www.worldtimebuddy.com/clock-widget

**Reply by bilogic** [reply]
Wow thanks, never saw that, but I couldn't slide the time around. The way I typically use it is to load the 2 timezones and slide to find a time when we are both awake.
